Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteLayerTreeDrawingArea.h

```diff
@@ -159,10 +159,12 @@ class RemoteLayerTreeDrawingArea : public DrawingArea, public WebCore::GraphicsL
         explicit BackingStoreFlusher(Ref<IPC::Connection>&&);
 
         Ref<IPC::Connection> m_connection;
+        Ref<IPC::Connection> protectedConnection() const { return m_connection.get(); }
         std::atomic<bool> m_hasPendingFlush { false };
     };
 
     Ref<RemoteLayerTreeContext> m_remoteLayerTreeContext;
+    Ref<RemoteLayerTreeContext> protectedRemoteLayerTreeContext() const;
```

Comment:
ditto.

Reply:
But also, can we make `m_remoteLayerTreeContext` const to avoid a protected function?